Actions/ReadSettings/ReadSettings.ps1

```diff
@@ -150,7 +156,7 @@ try {
                     $buildProjects = @($projects | Where-Object {
                         $project = $_
                         $buildProject = $false
-                        if (Test-Path -path (Join-Path $ENV:GITHUB_WORKSPACE "$project\.AL-Go\Settings.json")) {
+                        if (Test-Path -path (Join-Path $ENV:GITHUB_WORKSPACE "$project$([System.IO.Path]::DirectorySeparatorChar).AL-Go$([System.IO.Path]::DirectorySeparatorChar)settings.json")) {

```

Comment:
Are you sure the `$([System.IO.Path]::DirectorySeparatorChar).AL-Go$([System.IO.Path]::DirectorySeparatorChar)` is needed?

PS seems to be pretty smart about the slashes:
```
Join-Path 'a' 'b/c\d/e'
a\b\c\d\e
```

Reply:
I didn't know that - I have been through all occurances of DirectorySeperatorChar and modified to Join-Path